uat/src/main/java/com.aws.greengrass/CloudWatchSteps.java

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.aws.greengrass;
+
+import com.aws.greengrass.resources.CloudWatchLogStreamSpec;
+import com.aws.greengrass.resources.CloudWatchLogsLifecycle;
+import com.aws.greengrass.testing.features.WaitSteps;
+import com.aws.greengrass.testing.model.TestContext;
+import com.aws.greengrass.testing.modules.model.AWSResourcesContext;
+import com.aws.greengrass.testing.resources.AWSResources;
+import com.google.inject.Inject;
+import io.cucumber.guice.ScenarioScoped;
+import io.cucumber.java.en.Then;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import software.amazon.awssdk.services.cloudwatchlogs.model.LogStream;
+
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.List;
+import java.util.Locale;
+import java.util.TimeZone;
+import java.util.concurrent.TimeUnit;
+
+@ScenarioScoped
+public class CloudWatchSteps {
```

Comment:
🔪 Not needed here. This steps are log manager specific.

Reply:
removed